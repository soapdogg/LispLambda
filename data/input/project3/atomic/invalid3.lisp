(GREATER 33 (PLUS (TIM 0) 0))