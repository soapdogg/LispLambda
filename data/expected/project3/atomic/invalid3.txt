Error! Invalid CAR value: TIM